src/9.0/Siren.Interfaces/IUniverseFilter.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+namespace Siren.Interfaces;
+
+public interface IUniverseFilter
+{
```

Comment:
Make the return type `IEnumerable<T>`

Reply:
I would rather keep the `List<T>` because of the "Return the most specific type, accept the most generic type" principle, which is well described in [this article](https://enterprisecraftsmanship.com/posts/return-the-most-specific-type/#_collection_types).

Reply:
See the current solution and let me know. 👍